assets/local/shortcuts.js

```diff
@@ -54,7 +54,7 @@
 			// Focus the first element in popup
 			'keydown.ctrl_f': function () {
 				$('#editorFieldList').find('a:first-child').focus();
-				return false;
+				return true;
```

Comment:
By returning `true` here, you're allowing the browser default behaviour to occur, which is to open a search box (text field), and the browser gives that field focus, which renders the line above (`.focus()`) redundant.